install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```
@@ -242,8 +242,25 @@ spec:
                   docker kill "${container_id}" || true
                 done
               else
-                # COS-beta (with containerd). Some versions of COS have crictl in /home/kubernetes/bin.
-                for f in `find /var/lib/cni/networks/ -type f ! -name lock ! -name last_reserved_ip.0`; do PATH="${PATH}:/home/kubernetes/bin" crictl stopp "$(sed 's/\r//;1q' $f)" || true; done
+                if [ -d /var/lib/cni/networks/ ]; then
+                    echo "Sleeping for 60 seconds..."
+                    sleep 60
+                    echo "/var/lib/cni/networks/ found. Checking files for container-ids which are not managed by cilium ..."
```

```suggestion
                    echo "Checking /var/lib/cni/networks/"
```

I don't think we need to be more verbose here. We know what this code is doing

---

yep good idea.